plugin/src/utils/fetch-graphql.ts

```diff
@@ -488,6 +540,7 @@ if ( defined( 'GRAPHQL_REQUEST' ) && true === GRAPHQL_REQUEST ) {
   }
 
   reporter.panic({
+    id: CODES.BadResponse,
```

Comment:
Good catch on these! 😄

Reply:
ah yes, the gatsby definitions enforced this actually! the typescript error is hard to read, but i found that logging `panic({ context: {})` was invalid, wnd adding the ID made it valid. i think it has to do with the discrepancy in how typescript treats type unions vs function overloads. i'm making a little list of core typescript improvements to make for this!